src/common/SmartUrlInput.js

```diff
@@ -45,7 +49,7 @@ type Props = $ReadOnly<{|
    * it appears not to contain an explicit domain.
    */
   defaultDomain: string,
-  navigation: NavigationScreenProp<mixed>,
+  navigation: NavigationScreenProp<NavigationState>,
```

Comment:
I think this component may not belong in this commit -- see previous comment.

But also, the type here appears to vary from the pattern of the other changes.

Reply:
Ah, right; I just responded above, at https://github.com/zulip/zulip-mobile/pull/4273#discussion_r498003062.